org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/completion/AttributeValueCompletionResolver.java

```diff
@@ -69,4 +71,11 @@ private static MarkupContent getDocumentationForAttributeValue(CMElementDeclarat
 		return null;
 	}
 
+	@Override
+	protected void resolveCompletionItem(ICompletionItemResolverRequest request, CompletionItem toResolve,
+			DOMText text) {
+		// TODO Auto-generated method stub
```

Comment:
Add an implementation or change the comment to explain why an implementation isn't needed.